WcaOnRails/app/views/forum_topics/show.html.erb

```diff
@@ -0,0 +1,15 @@
+<!-- NOTE: This is meant for displaying old content of the phpBB forum. It is DEPRECATED! -->
+
+<div class="container">
+  <h2><%= @topic.topic_title %></h2>
+  <% @topic.forum_posts.sort_by { |post| -post.post_time }.each do |post| %>
```

Comment:
That's how it is done on the phpBB forum, but I don't have a strong opinion on this, so I will follow your thoughts =)